pennylane/collections.py

```diff
@@ -467,9 +520,29 @@ def evaluate(self, args, kwargs):
             list: the results from each QNode
         """
         results = []
+        parallel = kwargs.pop("parallel", False)
+        _scheduler = kwargs.pop("scheduler", "threads")
```

Comment:
Is there a case where we wouldn't want the scheduler to default to `threads`? I think it's fine but just wondering 🤔

Reply:
Don't think so! `processes` is slower and has more overhead, and `Client` only makes sense in a distributed set up